docs/source/quickstart.rst

```diff
@@ -1,88 +1,110 @@
 .. _quickstart:
 
-Quickstart
-==========
+Quickstart: Install MLflow, instrument code & view results in minutes
+======================================================================
 
-Installing MLflow
+
+In 15-20 minutes, you will:
```

Comment:
15-20 minutes sounds long. I think users can get through this in 10 minutes. How do you feel about saying "10 minutes"?

Reply:
Compromise "less than 15 minutes"? :-)